app/src/sharedTest/java/org/oppia/app/administratorcontrols/AdministratorControlsActivityTest.kt

```diff
@@ -187,6 +190,38 @@ class AdministratorControlsActivityTest {
     }
   }
 
+  @Test
+  fun testAdministratorControlsFragment_loadFragment_clickLogoutButton_displaysLogOutDialog() {
```

Comment:
testAdministratorControlsFragment_clickLogoutButton_displaysLogOutDialog

`loadFragment` is by default part of that activity and your test code is doing nothing to change that.

Reply:
Use consistent word:

`Logout` or `LogOut`. I would suggest use `Logout` everywhere.

Reply:
Done